pyvcloud/vcloudair.py

```diff
@@ -245,7 +245,8 @@ def get_service_groups(self):
         headers['Accept'] = "application/json;version=%s;class=com.vmware.vchs.billing.serviceGroups" % self.version
         self.response = Http.get(self.host + "/api/billing/service-groups", headers=headers, verify=self.verify, logger=self.logger)
         if self.response.history and self.response.history[-1]:
```

Comment:
I believe this workaround should not be necessary anymore, now that we are using 'vca.vmware.com' as the default endpoint for service type 'vca' (on-demand)